### PR TITLE
new icon: pypi (original, original-wordmark, plain, plain-wordmark)

### LINKED ISSUE #922: Icon request: Python Package Index (PyPI)
### About the icon

PyPI is the official third-party software repository for Python. A lot of people use Python therefore use PyPI on a daily basis, The main reason I need this icon for is to have a hyperlink to the PyPI page

## Links

Official Website: https://pypi.org/
Wikipedia Page: https://wikipedia.org/wiki/Python_Package_Index

## SVG sources

There are two logos, one large and one small
Large Logo: https://github.com/pypa/warehouse/blob/main/warehouse/static/images/logo-large.svg
Small Logo: https://github.com/pypa/warehouse/blob/main/warehouse/static/images/logo-small.svg

### Previews
Small logo:
Small Logo
Large logo:
Large Logo

### Additional information

An icon for npm exists so this would be helpful too

### PR DESCRIPTION
**Pull request by yeyeto2788**
## Double check these details before you open a PR

- [X] PR does not match another non-stale PR currently opened
- [X] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [X] PR's base is the `develop` branch.
- [X] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [X] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [X] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #922**

### CODE REVIEW

**Comment by yeyeto2788** [comment]
Sorry for the delay on this one guys! My bad. If there is anything I need to change or add let me know.

**Review by Snailedlt — CHANGES_REQUESTED** [review]
There's a few things that need to be changed, so I'll give a brief overview here. 
Read each comment for more info.

Some things that need to be fixed:
1. Move `devicon.json` object to correct line, so that the file is alphabetically sorted
1. Add altnames in `devicon.json`
1. Change to correct color in `devicon.json`
1. Remove *TM* from original-wordmark
1. Resize both icons to `viewbox="0 0 128 128"`

Optional (recommended):
1. Add `plain` and `plain-wordmark` versions
1. Add `line` and `line-wordmark` versions

File: icons/pypi/pypi-original.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...form="matrix(5.67725 0 0 5.65278 1130.868 -330.852)"/><path fill="#353564" stroke="#ccc" stroke-linejoin="bevel" stroke-width=".073" d="M-189.56 60.903v3.742l3.231-1.176v-3.742Zm0 0" transform="matrix(5.67725 0 0 5.65278 1130.868 -330.852)"/><path fill="#afafde" stroke="#ccc" stroke-linejoin="bevel" stroke-width=".073" d="m-189.56 64.645 3.184 1.16 3.231-1.177-3.184-1.159Zm0 0" transform="matrix(5.67725 0 0 5.65278 1130.868 -330.852)"/><path fill="#2f6491" stroke="#ccc" stroke-linejoin="bevel" stroke-width=".073" d="m-189.56 60.903 3.184 1.159 3.231-1.177-3.184-1.158Zm0 0" transform="matrix(5.67725 0 0 5.65278 1130.868 -330.852)"/><path fill="#3775a9" stroke="#ccc" stroke-linejoin="bevel" stroke-width=".073" d="M-186.376 62.062v3.743l3.231-1.177v-3.743Zm0 0" transform="matrix(5.67725 0 0 5.65278 1130.868 -330.852)"/><path fill="#2f6491" stroke="#ccc" stroke-linejoin="bevel" stroke-width=".073" d="m-189.56 60.903 3.184 1.159v3.743l-3.185-1.16Zm0 0" transform="matrix(5.67725 0 0 5.65278 1130.868 -330.852)"/><path fill="#fff" d="M83.137 26.766c0 2.062-1.578 4.304-3.524 5.011-1.945.703-3.523-.394-3.523-2.457 0-2.062 1.578-4.304 3.523-5.011 1.946-.707 3.524.394 3.524 2.457Zm0 0"/></svg>
```
[the remark this answers is not in the document]
Comment (yeyeto2788) [reply icons/pypi/pypi-original.svg]: The icon is not squared so the viewbox should be different right? I mean, the lead should be in the `width` having something like `0 0 128 112` or something similar right?

[the remark this answers is not in the document]
Comment (Snailedlt) [reply icons/pypi/pypi-original.svg]: No the viewbox should be the same, the icon just won't fill the entire viewbox :)

[the remark this answers is not in the document]
Comment (Snailedlt) [reply icons/pypi/pypi-original.svg]: Just make sure the icon is centered in the viewbox

[the remark this answers is not in the document]
Comment (yeyeto2788) [reply icons/pypi/pypi-original.svg]: Ohhhh got it then. I thought that was the actual dimensions of the icons. Changed applied.

[the remark this answers is not in the document]
Comment (Snailedlt) [reply icons/pypi/pypi-original.svg]: Nice, now you just need to center them vertically, and remove the horizontal padding
image

File: devicon.json [hunk 2]
```diff
@@ -7157,6 +7157,23 @@
...
+            "python"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark"
+            ],
+            "font": [
+                "plain"
+            ]
```
[the remark this answers is not in the document]
Comment (Snailedlt) [reply devicon.json]: @yeyeto2788 Seems like you need to have plain icons. To make them, just duplicate the original versions then do the following:
1. Unify paths
2. Remove the `fill` attribute

I also recommend making a line icon. See this guide for how to make one: https://github.com/devicons/devicon/issues/1306#issuecomment-1297625294

File: devicon.json [hunk 3]
```diff
@@ -7161,6 +7161,29 @@
         "color": "#21D789",
         "aliases": []
     },
+    {
+        "name": "pypi",
+        "altnames": [
+            "Python Package Index"
+        ],
+        "tags": [
+            "python"
```
Comment (Panquesito7) [inline devicon.json:7170]:
```suggestion
            "python",
            "package",
            "programming"
```

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🚀

**Review by Snailedlt — APPROVED** [review]
Great work fixing all of the issues in one go 💯
Thank you for the contribution! ✔️

**Pull request merged**